src/transformers/commands/add_new_model_like.py

```diff
@@ -761,7 +761,11 @@ def retrieve_info_for_model(model_type, frameworks: Optional[List[str]] = None):
         tokenizer_class = tokenizer_classes[0] if tokenizer_classes[0] is not None else tokenizer_classes[1]
     else:
         tokenizer_class = None
-    image_processor_class = auto_module.image_processing_auto.IMAGE_PROCESSOR_MAPPING_NAMES.get(model_type, None)
+    image_processor_classes = auto_module.image_processing_auto.IMAGE_PROCESSOR_MAPPING_NAMES.get(model_type, None)
+    if isinstance(image_processor_classes, tuple):
```

Comment:
I will always be a tuple. Instead we should check the length and similar to above check if the first class is not None